server/colors.js

```diff
@@ -0,0 +1,12 @@
+const colors = {
```

Comment:
The way you implement it is good, so you don't need to change it.

Just want to show you a different way that I think is preferable:
```js
const colors = {
   bright: {
      primary: "#EC7373",
      ...
   },
   dark: {
      primary: "#B95151",
      ...
   }
};
```

this way instead of doing this (pseudo code):
```js
setColor(theme == "bright" ? colors.brightPrimary : colors.darkPrimary);
```

you can just do this:
```js
setColor(colors[theme].primary);
```

which, in my opinion, is cleaner and clearer.
You can resolve this conversation right away.

Reply:
this is way better if there were more themes, noted